drafts/developer-guide/pages/data-privacy/index.adoc

```diff
@@ -0,0 +1,123 @@
+include::ROOT:partial$variables.adoc[]
+
+= Data privacy
+
+Data privacy allows us to configure and describe how and where sensitive data is stored on a module-field level.
+
+It expands the current data storing capabilities and adds features that enable {PRODUCT_NAME} administrators to have more control over data privacy and managing private data.
+
+=== *Defining Privacy Policies*.
+
+With information from data sources set on a specific data module, each record will be able to hold detailed information about the ownership and location(s) of particular data. 
+
+System Administrators will be obliged to provide a data storage location upon setting a Corteza data module. 
+
+{PRODUCT_NAME} will enable the selection of predefined data policies or new policies to describe the processing purpose and rules at a record field level.
+
+== Sensitivity Levels
+
+{PRODUCT_NAME} allows users to classify data by categorizing data assets based on their information sensitivity. By classifying data, organizations can determine two key things:
+
+Who should be authorized to access it?
+
+What protection policies to apply when storing and transferring it.
+
+There are  10 levels of data sensitivity with level 1 being the lowest and level 10 beng the highest.
```

Comment:
This is true but not quite correct; try to omit the fact we currently allow up to 10 levels.
The system, in reality, supports an unlimited amount of sensitivity levels. The UI currently limits to 10 because of the way it was designed.